test/test-script.js

```diff
@@ -510,34 +510,34 @@ describe('StrongZombie', function() {
   });
 
   it('should be alive', function() {
-    var tank = new StrongZombie(30, 30, 5);
     tank.isAlive.should.equal(true);
   });
-  it('should have a speedMultiplier attribute', function() {
-    var tank = new StrongZombie(30, 10, 25, 2);
+  it('should have a strengthMultiplier attribute', function() {
     tank.strengthMultiplier.should.equal(2);
   });
 
-  it('should have a buffSpeed method', function() {
-    var tank = new StrongZombie(30, 10, 25, 2);
+  it('should have a buffStrength method', function() {
     tank.buffStrength.should.exist;
   });
 
-  it('should multiply zombies speed by strengthMultipler when buffStrength called', function() {
-    var tank = new StrongZombie(30, 10, 25, 2);
+  it('should multiply zombies strength by strengthMultipler when buffStrength called', function() {
     var buffedStrength = tank.strength * tank.strengthMultiplier;
     tank.buffStrength();
     tank.strength.should.equal(buffedStrength);
   })
 }); // end StrongZombie specs
 
+var spitter;
```

Comment:
`var spitter`; should go inside `describe('RangedZombie'...`
just before `beforeEach`